src/case-worker-ref-data/case-worker-ref-data.routing.ts

```diff
@@ -0,0 +1,20 @@
+import { RouterModule, Routes } from '@angular/router';
+import { AuthGuard } from 'src/services/auth/auth.guard';
```

Comment:
This should be a relative path, not from "src".

Reply:
Yeah. The VS code auto code generator does this. Changed it now.